gradle/dependencies.gradle

```diff
@@ -82,7 +89,7 @@ versions += [
   reflections: "0.9.11",
   rocksDB: "5.15.10",
   scalatest: "3.0.5",
-  scoverage: "1.3.1",
+  scoverage: "1.4.0-M5",
```

Comment:
If we're going to depend on pre-release versions because of the scala version, we should probably leave a comment by the scala version with a note of everywhere that should be updated with it.

Reply:
One should be skeptic about merging pre-release dependency version into a trunk, options: 
 * leave appropriate comments and merge into trunk (this seems like a clean solution)
 * merge into a separate branch maybe (with 2.13 modifier)

Reply:
Done, I left related comment(s) that could be seen as an overzealous/overkill (will make them short if required or move them into commit message).